sql/firefox_nondesktop_exact_mau28_raw_v1.sql

```diff
@@ -14,6 +14,10 @@ SELECT
   COUNTIF(_inactive_days < 28) AS mau,
   COUNTIF(_inactive_days < 7) AS wau,
   COUNTIF(_inactive_days < 1) AS dau,
+  -- We hash client_ids into 20 buckets to aid in computing
+  -- confidence intervals for mau/wau/dau sums; the particular hash
+  -- function and number of buckets is subject to change in the future.
+  MOD(ABS(FARM_FINGERPRINT(client_id)), 20) AS id_bucket,
```

Comment:
I'm using `FARM_FINGERPRINT` because it's natively available in BQ, produces an integer that can be readily fed to `MOD`, and appears to be well tested to produce evenly distributed output. CRC32 is not available in BQ without resorting to a js UDF, but we will likely add support for CRC32-based sample_id as part of GCP pipeline metadata in coming months, so this will be an option down the line.